XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Simulation/Hands/SimulatedHandControllerDataProvider.cs

```diff
@@ -31,6 +35,10 @@ public SimulatedHandControllerDataProvider(string name, uint priority, Simulated
             }
 
             HandPoseAnimationSpeed = profile.HandPoseAnimationSpeed;
+            HandPhysicsEnabled = profile.HandPhysicsEnabled;
```

Comment:
I thought about adding this here but I wasn't sure if the simulated hands needed these options